README.md

```diff
@@ -1,2 +1,55 @@
 # storage-postgres
-A tailored version of PostgreSQL used as a Storage Plugin for FogLAMP
+This repository provides a tailored version of PostgreSQL used as a Storage Plugin for FogLAMP.
+
+Download a tarball with the prepared version of PostgreSQL available here:
+* For Intel [x86_64](https://s3.amazonaws.com/foglamp/plugins/storage/postgres/pgsql-foglamp-9.6_201608131-x86_64.tgz).
```

Comment:
what about armhf architecture?

Reply:
I have never seen it... the rasp pi is armv7l...
Where is it used?

Reply:
https://stackoverflow.com/questions/37790029/what-is-difference-between-arm64-and-armhf

With ubuntu-core on raspi3, architecture shows as armhf